src/services/api.js

```diff
@@ -212,6 +213,30 @@ SDKWrapper.prototype.verifyPhone = function (user: Object) {
     });
 };
 
+SDKWrapper.prototype.claimTokens = function ({ walletId, code }: ClaimTokenAction) {
+  return Promise.resolve()
+    // TODO update pillarWalletSdk
+    // .then(() => this.pillarWalletSdk.referral.claimTokens({ walletId, code }))
+    // TODO return just 200
+    .then(() => ({ responseStatus: 200, walletId, code }))
+    .catch((error) => {
+      const {
+        response: {
+          status,
+          data: { message } = {},
+        },
+      } = error;
+      Sentry.captureException({
+        error: 'Can\'t claim code',
```

Comment:
Maybe this message could be a bit more explicit to make life easier. 
Even providing info that it's the code for claiming referral tokens that fails, would help :)

Reply:
hmm then we pass more details from sever anyway like `message` to Sentry but changed to `Can\'t claim referral code'`

Reply:
👍